client/common/directives/ModelFieldWYSIWYG/ModelFieldWYSIWYG.js

```diff
@@ -22,21 +22,26 @@ angular.module('dashboard.directives.ModelFieldWYSIWYG', [
 })
 
 .directive('modelFieldWysiwygEdit', function($compile, $cookies, $timeout, $modal, Config, FileUploadService) {
-  function getTemplate() {
+  function getTemplate(scope) {
+    let fontSizeArr = [7, 10, 12, 14, 16, 18, 24];
+    let minSize = !(scope.options.fontSizeRange.min >= fontSizeArr.length) && (scope.options.fontSizeRange != undefined)
+    && !(scope.options.fontSizeRange.min <= 1) ? scope.options.fontSizeRange.min : 1;
+    let maxSize = !(scope.options.fontSizeRange.max >= fontSizeArr.length) && (scope.options.fontSizeRange != undefined)
+    && (scope.options.fontSizeRange.min <= scope.options.fontSizeRange.max)
+    && !(scope.options.fontSizeRange.max <= 1) ? scope.options.fontSizeRange.max : 7;
```

Comment:
@leolynn These checks aren't correct now. 1 and fontSizeArr.length are valid values, but you're treating them as invalid. I was saying you can either add parentheses OR invert </> to >=/<= and remove the negation, not both.

Also, please indent when you continue a single expression onto multiple lines (indent before &&)